cmd/version.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var version = "unknown"
```

Comment:
Commenting how this works in the code would be more useful than leaving it in the commit message.

Reply:
Added.